lib/net.c

```diff
@@ -23,6 +23,11 @@
 #include <arpa/inet.h>
 #include <linux/random.h>
 #include <sys/syscall.h>
+#include <sys/stat.h>
+#include <sys/types.h>
```

Comment:
Could you check that all these headers are needed? I think that `<sys/stat.h>` is not necessary.